src/dftbp/dftb/periodic.F90

```diff
@@ -460,16 +481,37 @@ subroutine updateNeighbourList(coord, img2CentCell, iCellVec, neigh, nAllAtom, c
     cutoff2 = cutoff**2
     nAllAtom = 0
 
+    parallel = .false.
+#:if WITH_MPI
+    parallel = present(env)
+#:endif
+
+    if (parallel) then
+#:if WITH_MPI
+      call distributeAtoms(env%mpi%nodeComm%rank, env%mpi%nodeComm%size, nAtom, &
+          & startAtom, endAtom, parallelSetupError)
+#:else
+      parallelSetupError = .true.
```

Comment:
Any reason not to just set
```suggestion
      parallel = .false.
```
as lines 496 - 498 are effectively just doing this?

Reply:
You are right. I just simplified the logic.